src/routes/user.js

```diff
@@ -1,15 +1,15 @@
 import { Router } from 'express'
 import { create, getById, getAll, updateById } from '../controllers/user.js'
 import {
-  validateAuthentication,
-  validateTeacherRole
+  validateAuthentication
+  // validateTeacherRole
 } from '../middleware/auth.js'
 
 const router = Router()
 
 router.post('/', create)
 router.get('/', validateAuthentication, getAll)
 router.get('/:id', validateAuthentication, getById)
-router.patch('/:id', validateAuthentication, validateTeacherRole, updateById)
-
+router.put('/', validateAuthentication, updateById)
+// removed validateTeacherRole middleware from the pathc method above ⬆
```

Comment:
You can remov ethis comment - usually commit history is used to capture the rationale for changes, the code itself doesn't need to document those.

Reply:
done